project/plugins.sbt

```diff
@@ -1,7 +1,8 @@
-addSbtPlugin("com.typesafe.sbt"  % "sbt-git"             % "0.8.5")
+addSbtPlugin("com.geirsson"      % "sbt-scalafmt"        % "0.4.8")
 addSbtPlugin("com.jsuereth"      % "sbt-pgp"             % "1.0.0")
-addSbtPlugin("de.heikoseeberger" % "sbt-header"          % "1.5.1")
-addSbtPlugin("org.scalariform"   % "sbt-scalariform"     % "1.6.0")
+addSbtPlugin("com.typesafe.sbt"  % "sbt-git"             % "0.8.5")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-multi-jvm"       % "0.3.11")
+addSbtPlugin("de.heikoseeberger" % "sbt-header"          % "1.5.1")
 addSbtPlugin("me.lessis"         % "bintray-sbt"         % "0.3.0")
+addSbtPlugin("org.scalariform"   % "sbt-scalariform"     % "1.6.0")
```

Comment:
In this case, the best is to be aligned with constructR. Having said that, I prefer `scalafmt` ;)